tests/network/provider_migration/test_ip_persistence.py

```
@@ -1,18 +1,66 @@
+@pytest.mark.polarion("CNV-12208")
+def test_network_data_preserved_after_vm_import(source_vm_network_data, imported_cudn_vm):
+    source_vm_mac, source_vm_ip = source_vm_network_data
+    target_vm_iface = lookup_iface_status(
+        vm=imported_cudn_vm, iface_name=lookup_primary_network(vm=imported_cudn_vm).name
+    )
+    target_vm_mac, target_vm_ip = target_vm_iface.get("mac", None), target_vm_iface.get(IP_ADDRESS, None)
+
+    assert source_vm_mac == target_vm_mac and source_vm_ip == target_vm_ip, (
+        f"The network data was not preserved during VM import. Expected: MAC={source_vm_mac}, IP={source_vm_ip}. "
+        f"Got: MAC={target_vm_mac}, IP={target_vm_ip}."
     )
```

Ruff (0.14.10)

Assertion should be broken down into multiple parts

(PT018)